content/guides/component-testing/slots-angular.md

```diff
@@ -0,0 +1,159 @@
+---
+title: Testing Angular Components with Slots
+---
+
+Content projection is a pattern in which you insert, or **project**, the content
+you want to use inside another component. In angular there are three types
+content projection, `single-slot`, `multi-slot`, and `conditional`.
+
+Like props and events, slots are part of the component's public API.
+
+## Single-slot Content Projection
+
+The most basic form of content projection is known as **single-slot content
+projection**. This refers to creating a component into which you can project one
+component using `<ng-content></ng-content>`. You can learn more about
+Single-slot content projection at
+[https://angular.io/guide/content-projection#single-slot](https://angular.io/guide/content-projection#single-slot)
+
+Below is a simple ButtonComponent that is using **single-slot content
+projection**.
+
+<code-block label="button.component.ts" active>
+
+```ts
+import { Component } from '@angular/core';
+
+@Component({
+    selector: 'app-button',
+    template: `
+        <button>
+            <ng-content></ng-content>
+        </button>
+    `
+})
+export class ButtonComponent {...}
+```
+
+</code-block>
+
+Now we create a **WrapperComponent** in our spec so we use to simulate passing
```

Comment:
I would ditch the wrapper component example and just use the template example. I would only show a wrapper example if theres a strong use case to do so that is better alternative to template.